packages/focusvisible/src/useFocusVisible.js

```diff
@@ -0,0 +1,255 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+/**
+ * Inspired by the [WICG/focus-visible](https://github.com/WICG/focus-visible)
```

Comment:
This is more than inspired right it's essentially the hookification of the polyfill. Can we preserve the comments from the polyfill too, shame we can't use the polyfill directly in this hook.

Reply:
I've updated with consistent commenting.